protostar/configuration_file/configuration_file.py

```diff
@@ -86,6 +86,17 @@ def set_command_names_provider(
     ):
         pass
 
+    @staticmethod
+    def create_appending_cairo_path_suggestion() -> str:
+        return "\n".join(
+            [
+                "You may want to add your new library's path to the 'cairo_path' "
+                "section of your configuration file (usually `protostar.toml`).",
```

Comment:
```suggestion
                "section of your configuration file.",
```
We can relatively easy get the name of the configuration file (but this method can't be static). We should either print the exact name or remove the guess.
Instead of the file name, a relative file path could be printed so that the user can `cmd+click` it.